packages/app-desktop/gui/utils/NoteListUtils.test.ts

```diff
@@ -0,0 +1,207 @@
+import NoteListUtils from './NoteListUtils';
+import KeymapService from '@joplin/lib/services/KeymapService';
+import menuCommandNames from '../menuCommandNames';
+import { MenuItem as MenuItemType } from '@joplin/lib/services/commands/MenuUtils';
+import initializeCommandService from '../../utils/initializeCommandService';
+import { createAppDefaultWindowState } from '../../app.reducer';
+
+jest.mock('../../services/bridge', () => ({
+	__esModule: true,
+	default: () => ({
+		MenuItem: class MenuItem {
+			public value: MenuItemType;
+			public constructor(value: MenuItemType) {
+				this.value = value;
+			}
+		},
+		Menu: class MockMenu {
+			public append = jest.fn();
```

Comment:
Thank you for adding tests!

**Optional suggestion**:
I could see the `toHaveBeenNthCalledWith`s being somewhat difficult to maintain when adding/removing context menu items in the future.

```suggestion
			public items: MenuItemType[] = [];
			public append(item: MenuItemType) {
				this.items.push(item);
			}
```

Then later, it would be possible to use `.toEqual` (or possibly snapshots) instead of `.toHaveBeenNthCalledWith`.

Reply:
Makes sense! Thanks for the suggestion. I'm using snapshots now.